### ISSUE TITLE
Dashboard left-side navigation should be a list

### ISSUE THREAD

**Issue by aitchiss** [forem/forem#14841]
**Describe the bug**

The left-side navigation options on the dashboard page (desktop view) are correctly wrapped in a `<nav>` element but they should also be wrapped in a `<ul>`, each as individual `<li>` list items.

This helps screen reader users navigate between the items, knowing how many options are in the list and their current position in that list.

**To Reproduce**

- Using a screen reader visit the left-hand navigation on the `/dashboard` page
- Notice the number of items in the navigation is not announced

**Expected behavior**

- Items are wrapped in a list and the number of items is announced when using a screen reader

**Screenshots**

![screenshot of the navigation panel](https://user-images.githubusercontent.com/20773163/135101251-8bbf9a50-d5b7-4296-9951-599087cd9681.png)

**Comment by secondl1ght** [comment]
Hello, I would like to give this issue a try for my first Hacktoberfest PR. I will take a crack at it tomorrow, just wanted to make a comment in case more than one person was looking at this one. Thanks!

**Comment by aitchiss** [comment]
I've assigned it to you @secondl1ght - thanks for taking it on!

**Comment by wweverma1** [comment]
@aitchiss  Hello, Is this Issue closed or can I raise a PR for it?

**Comment by cmgorton** [comment]
@wweverma1 This issue has been assigned to @secondl1ght .

**Comment by secondl1ght** [comment]
I have submitted a PR for this issue, #14887. Thanks!
